gui/slick/interfaces/default/home.mako

```diff
@@ -814,6 +818,15 @@ $(document).ready(function(){
             </script>
         </td>
 
+        <% show_size = 0 %>
+        % try:
+            <% show_size = sickbeard.helpers.get_size(curShow.location) %>
+        % except sickbeard.exceptions.ShowDirNotFoundException:
+            <% show_size = sickbeard.helpers.get_size(curShow._location) %>
+        % endtry
+
```

Comment:
I wasn't really sure what I was supposed to do :/
https://github.com/MGaetan89/SickRage/commit/16b0a4f2421a8b0f32fb9d14a6a498e09e8abd89
- In `home.mako` I just request the size of the show, and delegate the check if the folder exists to `get_size`
- In `helpers.py#get_size()` I check if the path exists before looping through its children